packages/networks/signer.ts

```diff
@@ -21,6 +21,10 @@ import {
   keplrChainInfoFromNetworkInfo,
   mustGetCosmosNetwork,
 } from "@/networks/index";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
@n0izn0iz 
I could relocate the slices, but I prefer the current location. Should I move it to override the linter?`

Reply:
the slice is good in store package, would be better to pass the wallet slice to the function and use selector hook to get the data or something similar so the networks package doesn't depend on store